layouts/partials/docs/toc.html

```diff
@@ -10,6 +10,11 @@ <h4 class="no-anchor">
         </h4>
         {{ template "toc" (dict "page" . "menu" .Site.Menus.quickstart) }}
 
+        <h4 class="no-anchor">
+            <a href="{{ relref . "/docs/console" }}">Pulumi Cloud Console</a>
```

Comment:
Yeah, we may also want to make the fonts smaller on the left nav. And it took me a few days to realize you could click on the top-level nav items like "Get Started" and Reference" . I wonder if we should have a `caret-down` icon next to them? And either move the "index" content under an Overview page for the top-level item, or make it clearer that it's clickable and serves as the index or landing page for that category. @cnunciato - I'm sure you have thoughts around this, too.

Reply:
For the short-term, it may make the most sense to put it at the bottom. So no push back from me. But I think that the end-state should have Pulumi Cloud Console at the top and _maybe_ under "Get Started". 

The core issue is that "Pulumi" may mean two different things: the code-focused framework for creating cloud resources (e.g. CLI, code examples) or it is the SaaS application for tracking resource state (e.g. console, screenshots). Those are two very different things (and perhaps serving two different use-cases / situations).

I can see us eventually moving the Cloud Console docs to a separate section of the website entirely, perhaps splitting "product documentation" out from "library/package API reference". But like I said, we're probably still a ways away from that. So it's more important to just flesh out the content for the Cloud Console and put it _somewhere_.

We can find a better home later, when our L2 nav becomes even more crowded and we need to vote something off the island.

Reply:
> or make it clearer that it's clickable

💯 yep &mdash; it's been on my mental list, but I haven't had a chance to propose an alternative yet. Will work on that.

@chrsmith This is great. I finally know where all this stuff is, now. 😄 🍹